### ISSUE TITLE
Unexpected click outline

### ISSUE THREAD

**Issue by karmatosed** [WordPress/twentytwenty#598]
I was testing the menu and got this unexpected image:

![image](https://user-images.githubusercontent.com/253067/65799381-e8808d80-e16b-11e9-9364-1ffde2ab9c39.png)

After digging I also get on desktop:

![image](https://user-images.githubusercontent.com/253067/65799405-f59d7c80-e16b-11e9-9956-3307f7e73818.png)

**Comment by karmatosed** [comment]
It seems to not just be on menus. I found it also on search:

![image](https://user-images.githubusercontent.com/253067/65799623-7e1c1d00-e16c-11e9-86aa-9a6ea966abf4.png)

**Comment by afercia** [comment]
This appears to be the browser's native focus indication.

Are you testing with the Opera browser? (I see the outline is yellow, which is only used by Opera I think 🙂 )

Not a bug _per se_  I'd say but maybe the focus indication could be improved. Two consderations:

1
The particular shape of the focus on Search and Menu with that sort of protruding rectangle is given by the `width: 10rem;` applied to the button text:

Opera:

<img width="501" alt="Screenshot 2019-09-28 at 12 05 25" src="https://user-images.githubusercontent.com/1682452/65814974-7058ad00-e1e9-11e9-9992-20264d8caa20.png">
<img width="492" alt="Screenshot 2019-09-28 at 12 05 34" src="https://user-images.githubusercontent.com/1682452/65814975-7058ad00-e1e9-11e9-8c81-7409a5a4c0cd.png">

Chrome:

<img width="596" alt="Screenshot 2019-09-28 at 12 08 23" src="https://user-images.githubusercontent.com/1682452/65814986-8f573f00-e1e9-11e9-8fbe-303eb0791d22.png">

I'd tend to think `width: 10rem;` is a "magic number" that should be avoided with some refactoring of the CSS.

2
Worth reminding the browsers native focus style differs across browsers. In Firefox the native style is very light and a custom style would be recommended.

Firefox:

<img width="729" alt="Screenshot 2019-09-28 at 12 09 49" src="https://user-images.githubusercontent.com/1682452/65815008-dd6c4280-e1e9-11e9-9ab3-df7898e6a35e.png">
<img width="707" alt="Screenshot 2019-09-28 at 12 09 59" src="https://user-images.githubusercontent.com/1682452/65815009-dd6c4280-e1e9-11e9-8f6f-765a33b7a3e9.png">

**Comment by karmatosed** [comment]
So the issue seems to be then that it needs a style over being default?

**Comment by carolinan** [comment]
I would like it to be switched to the same style that was agreed on during the accessibility meeting. _a solid outline and adding and outline offset of 2 pixels_
https://make.wordpress.org/accessibility/2019/09/29/wordpress-accessibility-meeting-notes-for-27-sept-2019/

Example gif: https://slack-files.com/T024MFP4J-FNTF6LTE0-a521c3312a

**Comment by carolinan** [comment]
The original issue regarding the **shapes** has been fixed.

However now there is no visible focus on the search modal input field when using Firefox (on PC):
![search-modal](https://user-images.githubusercontent.com/7422055/66593223-e0c2df00-eb96-11e9-9cca-f22d44d28289.png)
